server/api/jellyfin.ts

```diff
@@ -437,6 +454,51 @@ class JellyfinAPI extends ExternalAPI {
       throw new ApiError(e.response?.status, ApiErrorCode.InvalidAuthToken);
     }
   }
+
+  public async deleteUserDevice(
+    userId: string,
+    deviceId: string
+  ): Promise<void> {
```

Comment:
Couldn't you just do `await this.axios.delete('/Devices', { params: { Id: deviceId } });` and check the result code of this delete instead of all this?